packages/zoe/package.json

```diff
@@ -11,9 +11,9 @@
   },
   "scripts": {
     "build": "yarn build-zcfBundle",
-    "test": "yarn test:node && yarn test:xs-unit",
-    "test:node": "ava --verbose",
+    "test": "ava --verbose",
     "test:nyc": "nyc ava",
+    "test:xs": "yarn test:xs-unit",
     "test:xs-unit": "ava-xs --verbose",
     "test:xs-unit-debug": "ava-xs --debug",
```

Comment:
this reminds me... arg parsing limitations in `ava-xs` that motivated `test:xs-unit-debug` are gone, so this should go. I can't remember whether the `ava-xs` fix is on master or a pending branch, though.